xml/System.Security.Cryptography.X509Certificates/X509Certificate.xml

```diff
@@ -2281,6 +2290,8 @@
         <returns>The public key for the X.509 certificate as an array of bytes.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[  
+## Remarks
+This value corresponds to the contents of the `subjectPublicKey` field of the SubjectPublicKeyInfo data within the certificate.
```

Comment:
Note to anyone else reviewing this PR: "SubjectPublicKeyInfo" is not an API, so I think it's ok that we show it like that. See [the other existing examples](https://github.com/dotnet/dotnet-api-docs/search?l=XML&q=SubjectPublicKeyInfo).